### PR TITLE
get remote single post by post id

### PR DESCRIPTION
**Pull request by PENGYUXIONG**

### CODE REVIEW

File: backend/api/services/nodeServices.py [hunk 1]
```diff
@@ -173,8 +173,22 @@ def getRemotePosts(request, **kwargs):
     Request a single Remote Post
     """
     @staticmethod
-    def getSingleRemotePost():
-        pass
+    def getSingleRemotePost(request):
+        # get external node
+        extNodeReq = request.headers.get('X-Request-Host')
+        extNode = Node.objects.get(host_url=extNodeReq)
```
Comment (AnkushSharma2698) [inline backend/api/services/nodeServices.py:179]: maybe have an error check in case the node doesnt exist?

**Comment by AnkushSharma2698** [comment]
Overall Looks good to me

**Review by AnkushSharma2698 — APPROVED** [review]

File: backend/api/services/nodeServices.py (hunk 1 above)
Comment (PENGYUXIONG) [reply backend/api/services/nodeServices.py:179]: will return `None` now if extNode does not exist

Comment (AnkushSharma2698) [reply backend/api/services/nodeServices.py:179]: Good stuff, good to merge I would say

**Pull request merged**